src/components/Comments.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsPerArticle } from "../../utils/api";
+import Collapsible from "./Collapsible";
+
+function Comments() {
+    const [isLoading, setIsLoading] = useState(true);
+    const [comments, setComments] = useState(null);
+    const { item_id } = useParams();
+
+    useEffect(() => {
+        setIsLoading(true);
+        getCommentsPerArticle(item_id)
+            .then((comments) => {
+                setComments(comments);
+                setIsLoading(false);
+            })
+    }, [item_id]);
+
+    if (isLoading) {
+        return <p className="Loading">Loading...</p>;
+    }
+
+    if (!comments) {
+        return <p>No comments, be the first!</p>;
+    }
+
+    return (
+        <Collapsible title="Toggle Comments">
+            <ul className="Comments_list">
+                {comments.map((comment) => (
+                    <li className="Comment_card" key={comment.comment_id}>
+                        <h3>{comment.author}</h3>
+                        <p>{comment.body}</p>
+                        <p>Votes: {comment.votes} | ID: {comment.comment_id}</p>
```

Comment:
The user doesn't need to know what the comment ID is - that's more for the dev behind the scenes, so there's no need to display it to them :)